### ISSUE TITLE
Styled Components Missing.

### ISSUE THREAD

**Issue by AdamAnSubtractM** [nickjuntilla/safe-sale#1]
This is missing Styled Components.

**Comment by nickjuntilla** [comment]
@AdamAnSubtractM Styled components are inferior to every other modularization of CSS. They encourage 1:1 abstraction and create a Rube Goldberg machine for doing simple things like adding conditional styles. They create new memory space and objects where none are needed. I couldn't imagine a worse idea for a library. Style-JSX is better. CSS Modules are better. Even plain CSS is better. Styled components are a symptom of when front end developers have too much time on their hands and start creating work for themselves which ultimately leads to less productivity for the business. They also don't play well with typescript and the list goes on and on. You create a translation layer between css syntax and javascript keys. Do you need more reasons why it's a horrible idea?

**Comment by aknee-dmi** [comment]
Actually, Styled Components is amazing with Typescript! I suggest you read the docs if you are unsure how to utilize correctly.

**Comment by nickjuntilla** [comment]
@AdamAnSubtractM Yes you can use it with typescript if you want to code even slower and create even more boilerplate and redundancy.
